docs/build/linux-ubuntu.md

```diff
@@ -98,4 +98,51 @@ cd keymanapp/keyman
 docker run -it --rm -v $(pwd):/home/build/src/keyman -u root -w /home/build/src/keyman keymanapp/keyman-linux-builder:latest bash -c "cd linux && make reconf && make fullbuild && make install"
 ```
 
+## Keyman for Android
 
+**Dependencies**:
+* [Base](#base-dependencies)
+* [Web](./windows#web-dependencies)
+
+**Additional requirements**:
+* Android SDK
+* [Android Studio](https://developer.android.com/studio/install#linux)
+* Gradle
+* Maven
+* OpenJDK 11 (for Keyman 17.0+)
+
+
+    Run Android Studio once after installation to install additional components
+  such as emulator images and SDK updates.
+
+**Required environment variables**:
+* [`JAVA_HOME`](#java_home)
+
+Building:
+* [Building Keyman for Android](../../android/README.md)
+
+## Prerequisites
+
+Many dependencies are only required for specific projects.
+
+### Base Dependencies
+
+**Environment variables**:
+
+
+## Notes on Environment Variables
+
+### JAVA_HOME
+
+This environment variable tells Gradle what version of Java to use for building Keyman for Android.
+OpenJDK 11 is used for master.
+It's recommended to install both and set the environment variables to:
+```bash
+EXPORT JAVA_HOME_11="path to OpenJDK 11"
+EXPORT JAVA_HOME="$JAVA_HOME_11"
+```
```

Comment:
I don't think we should be setting `JAVA_HOME_11` for most usages. Only for those working in both Keyman <=16 and Keyman 17+...